### PR TITLE
factories: add setLanguage factory

### PR DESCRIPTION
**Pull request by andrewm89**
* Add angular factory to set gettext language
* Add bundles.py to compile the factory
  (addresses https://github.com/inveniosoftware/invenio-i18n/issues/34)

Signed-off-by: Andrew McCracken <andrew@tind.io>

### CODE REVIEW

File: invenio_i18n/static/js/angularLangSelector.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "node_modules/angular/angular",
+    "node_modules/angular-gettext/dist/angular-gettext"
+], function() {
+    angular.module("langSelector", ["gettext"])
+        .factory("setLanguage", ["gettextCatalog", function(gettextCatalog){
+            function setCurrentLanguage(lang) {
+                gettextCatalog.setCurrentLanguage(lang);
+            }
+        }]);
+});
```
Comment (jirikuncar) [inline invenio_i18n/static/js/angularLangSelector.js]: `\n\0`

File: invenio_i18n/bundles.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+from __future__ import absolute_import, print_function
+
+from invenio_assets import NpmBundle, RequireJSFilter
+
+js = NpmBundle(
+    "js/angularLangSelector.js",
+    filters=RequireJSFilter(),
+    output='gen/i18n.%(version)s.js',
+    npm={
+        "angular-gettext": "latest"
```
Comment (jirikuncar) [inline invenio_i18n/bundles.py]: `s/$/,/`

**Comment by drjova** [comment]
LGTM

**Comment by jirikuncar** [comment]
LGTM

**Pull request merged**